src/main/java/org/czekalski/petapiintegration/resourceserver/model/Animal.java

```diff
@@ -0,0 +1,76 @@
+package org.czekalski.petapiintegration.resourceserver.model;
+
+public class Animal {
+    private Integer id;
+    private String name;
+    private String breed;
+    private String age;
+    private String gender;
+
+
+    private Animal(AnimalBuilder animalBuilder) {
```

Comment:
Oj tak nie działa builder. Ogólnie są akceptowane dwa rozwiązania. Jedno to, prywatny pusty konstruktor i builder ustawia wartości bezpośrednio w polach w nowej instancji utworzonej w metodzie build i ją zwraca. A drugie to wypełnienie konstruktora już wartościami.

Reply:
poprawiono: 96619b9b46fa538703ca171c44c795010b4ffceb